extrinsic_imu2base_quat_xyzw_xyz: [0.0, 0.0, 0.0, 1.0, 0.05, 0.0, -0.12]
extrinsic_lidar2base_quat_xyzw_xyz: [0.0, 0.0, 0.0, 1.0, 0.00, 0.0, 0.0]
double_downsample: false
initialization_phase: true
timestamps:
  force_relative: true
